lib/babe/babe.go

```diff
@@ -169,10 +169,6 @@ func NewService(cfg *ServiceConfig) (*Service, error) {
 		return nil, errNilEpochState
 	}
 
-	if cfg.BlockImportHandler == nil {
-		return nil, errNilBlockImportHandler
-	}
-
```

Comment:
Note this is removed to allow to have `nil` block import handler in test code.
Anyway imo this check shouldn't be there, and we should panic if we try to pass a nil handler.